source/Calamari.Terraform/Program.cs

```diff
@@ -1,13 +1,14 @@
 using System;
 using Calamari.Commands.Support;
+using Calamari.Common;
 using Calamari.Plumbing;
 
 namespace Calamari.Terraform
 {
     public class Program : CalamariFlavourProgram
     {
 
-        public Program(ILog log) : base(log)
+        Program(ILog log) : base(log)
```

Comment:
I'm not sure why the lines below are not coming up in the diff as they are not on `master`, however pretty much all of that should be done in the base class.